packages/spl-token-collective/src/index.ts

```diff
@@ -639,26 +642,48 @@ export class SplTokenCollective extends AnchorSdk<SplTokenCollectiveIDL> {
       throw new Error("Collective already exists");
     }
 
-    instructions.push(
-      await this.instruction.initializeCollectiveV0(
-        // @ts-ignore
-        {
-          authority: authority ? authority : null,
-          bumpSeed: collectiveBump,
-          config: toIdlConfig(config),
-        },
-        {
-          accounts: {
-            collective,
-            mint: mint!,
-            mintAuthority: mintAuthority!,
-            payer,
-            systemProgram: SystemProgram.programId,
-            rent: SYSVAR_RENT_PUBKEY,
+    if (socialTokenRef) {
```

Comment:
You should be able to derive the socialTokenRef from the mint. So even if they don't pass this variable, you should still check to see if the token is a social token.